tailwind.config.cjs

```diff
@@ -33,11 +33,11 @@ module.exports = {
       },
       width: {
         leftnavbar: '48px',
-        leftsidebar: '225px',
+        leftsidebar: '170px',
       },
       minWidth: {
         leftnavbar: '48px',
-        leftsidebar: '225px',
+        leftsidebar: '170px',
```

Comment:
not directly related to this PR, but it makes me feeling that maybe the dimension of the sidebar should not be part of the tailwind configuration but of the component or the core (like the color-registry)

where the value are then being injected into the CSS

because if we change the value there, probably that AI lab would have to change it in their fork of the tailwind config while we might think that the width is set by Podman Desktop

Reply:
@benoitf should AI lab plan to change it in their extensions too / implement the navigation bar? i'm assuming it's a separate svelte file for them?

would it be suitable to include the pixel width in the color-registry file so it's easily retrievable like everything else? unsure if this should be a separate PR or not

Reply:
(it's just thoughts there, that I wanted to share but it won't hold the PR)

Reply:
but yes it's defined there:
https://github.com/containers/podman-desktop-extension-ai-lab/blob/main/packages/frontend/tailwind.config.cjs#L51-L57

but still having the item being defined there https://github.com/containers/podman-desktop-extension-ai-lab/blob/main/packages/frontend/src/lib/Navigation.svelte#L5

so probably enhancement need to be done on the component's side

Reply:
Ahh that makes sense. Let's open up an issue regarding this so we can help standardize this. I know we've been discussing webviews as well as subnavigation, so I'm unsure what the result of that discussion may impact this either.